build/rust/Cargo.lock

```diff
@@ -1251,9 +1250,9 @@ checksum = "830d08ce1d1d941e6b30645f1a0eb5643013d835ce3779a5fc208261dbe10f55"
 
 [[package]]
 name = "libc"
-version = "0.2.139"
+version = "0.2.141"
```

Comment:
Whoops sorry, I was playing around with bumping the package versions because one of my original theories was that there was a regex somewhere which was taking a long time to execute - I'll revert this.